src/Features/Core/Portable/RemoveUnusedParametersAndValues/AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer.SymbolStartAnalyzer.BlockAnalyzer.cs

```diff
@@ -137,8 +137,9 @@ private void AnalyzeExpressionStatement(OperationAnalysisContext context)
 
                     // Bail out cases for report unused expression value:
 
-                    //  1. Null type and void returning method invocations: no value being dropped here.
+                    //  1. Null type, error type and void returning method invocations: no value being dropped here.
                     if (value.Type == null ||
+                        value.Type.IsErrorType() ||
```

Comment:
we have IsNullOrErrorType as a helper somewhere i think :)

Reply:
Ah thanks! Will keep in mind for now on.